pyprland/plugins/scratchpads/__init__.py

```diff
@@ -531,13 +531,15 @@ async def run_show(self, uid) -> None:
     async def _show_transition(self, item, monitor, was_alive):
         "perfoms the transition to visible state"
         animation_type = item.conf.get("animation", "").lower()
-        wrkspc = monitor["activeWorkspace"]["name"]
+        wrkspc = monitor["specialWorkspace"]["name"]
+        if wrkspc == "":
```

Comment:
If we aren't in a special workspace, the name of it will be empty and id will be 0.